src/test/java/io/stargate/sgv2/jsonapi/api/v1/AbstractKeyspaceIntegrationTestBase.java

```diff
@@ -234,13 +234,16 @@ public static void checkShouldAbsentMetrics(String commandName) {
 
   public static void checkDriverMetricsTenantId() {
     String metrics = given().when().get("/metrics").then().statusCode(200).extract().asString();
+    // Example line
+    // session_cql_requests_seconds_bucket{module="sgv2-jsonapi",session="3ee89833-3f8c-4140-841f-59ae95bc48e0",le="0.001"} 0.0
+
     Optional<String> sessionLevelDriverMetricTenantId =
         metrics
             .lines()
             .filter(
                 line ->
                     line.startsWith("session_cql_requests_seconds_bucket")
-                        && line.contains("tenant"))
+                        && line.contains("session="))
```

Comment:
Does this mean metric tag changes? Is that ok for production deployments?

Reply:
yes, this is for the metrics from the driver, I could not see any panels that included those.